vision/pipeline.py

```diff
@@ -88,44 +93,92 @@ def run(self, prev_state):
             state = self.flight_communication.get_nowait()
         except Empty:
             state = prev_state
-
         ##
+
         bboxes = []
 
+        flags = {}
+
+        state = "module_detection"
```

Comment:
Should remove this state assignment here since it would reassign it every time to module detection

Reply:
For testing purposes, forgot to take it out-- good catch